mono/mini/exceptions-arm.c

```diff
@@ -14,7 +14,7 @@
 #include <string.h>
 
 #ifndef MONO_CROSS_COMPILE
-#ifdef HAVE_ASM_SIGCONTEXT_H
+#if defined(HAVE_ASM_SIGCONTEXT_H) && defined(__GLIBC__)
 #include <asm/sigcontext.h>
```

Comment:
In [mono-sigcontext.h#L5-L7](https://github.com/mono/mono/blob/438c8f378c1f1000bdc82f0156fa153eaf9138f2/mono/utils/mono-sigcontext.h#L5-L7) we seem to only include the file on PLATFORM_ANDROID, maybe the same needs to be done here instead? I'm not familiar with the code though, @tritao is the expert :)